cmd/publicize/server_test.go

```diff
@@ -174,7 +178,42 @@ func TestCheckPrerequisites(t *testing.T) {
 func TestMergeAndPushToRemote(t *testing.T) {
 	publicOrg, publicRepo := "openshift", "test"
 	privateOrg, privateRepo := "openshift-priv", "test"
-
+	fixedTime := time.Now()
+	fixedTimeUnix := fixedTime.Unix()
+	makeRepo := func(localgit *localgit.LocalGit, org, repo string, init func() error) error {
```

Comment:
Init is always nil or did I miss it? Also, lets add the explanation from the PR as a godoc here :)

Reply:
It's passed the `privateGitRepo`/`publicGitRepo` members of `testCases`, which can be nil but sometimes aren't.

Reply:
The PR description is a direct copy of the commit message.  Do you think it's necessary to add a comment as well?

Reply:
Yeah, because the commit message is not visible when just reading the code and backtracking the commit that introduced it to understand it through that commits commit message is tedious

Reply:
Added.